lib/broccoli/ember-app.js

```diff
@@ -64,13 +65,9 @@ module.exports = EmberApp;
   @param {Object} options Configuration options
 */
 function EmberApp() {
-  var args = [];
+  var args = map(arguments, function(item) { return item });
```

Comment:
Let's rollback to the old implementation but extract them into separate function.

Let me know if it's better.